### PR TITLE
configure private parking space

### PR DESCRIPTION
**Pull request by jkcny**
Featues included in this Pull Request:
- Update add_spot form for Private Parking Lot
- Configure the Private Parking Lot specific infoWindow for the google map markers

### CODE REVIEW

File: map/templates/map/add_spot.html [hunk 1]
```diff
@@ -61,6 +66,10 @@ <h3 class="text-center">New Post</h3>
             </option>
           </select>
         </div>
+        <div class="form-group py-1">
```
Comment (Zeroays0) [inline map/templates/map/add_spot.html]: Shows up even if the owner is not verified

Should this be conditionally rendered only if the owner is verified for Private Parking?

Comment (Zeroays0) [reply map/templates/map/add_spot.html]: Also, can we add validation to the Capacity Input field so it must be >= 1?

**Review by Zeroays0 — COMMENTED** [review]
See my comments.

TLDR ; 
Add validation to Vehicle Spaces Capacity (>= 1)
Conditionally render Vehicle Spaces Capacity IF Private Spot ONLY?

File: map/templates/map/parking.html [hunk 2]
```diff
@@ -764,14 +796,27 @@ <h4 class="card-title">${cur.title || "No title"}</h4>
...
-      (spot.type === "Street" || (spot.type === "Business" && userIsOwnerOfBusiness)) &&
+      (spot.type === "Street" || ((spot.type === "Business" || spot.type === "Private") && userIsOwnerOfBusiness)) &&
       isUserAuthenticated
     ) {
+      
+      if (spot.type === "Private") {
+        return (
+        '<div class="form-group mb-3">' +
+        '<label for="occupancy-status">Available Vehicle Spaces:</label>' +
+        `<input type="number" class="form-control" id="available-vehicle-spaces-input" name="available-vehicle-spaces" min="0" max="${spot.vehicle_spaces_capacity}" step="1">` +
```
Comment (Zeroays0) [inline map/templates/map/parking.html]: Should the input field for "Available Vehicle Spaces" be populated?  I see the Occupancy Percent is updated but nothing in the input field when the spot info window is loaded.

Comment (Zeroays0) [reply map/templates/map/parking.html]: Also users can enter negative numbers or > capacity values manually

**Review by tmaricic1 — APPROVED** [review]
LGTM

**Comment by Andrewhg414** [comment]
There was a bug that was not caught here. Create Spot had required fields when done for a street spot. Jimmy has written a fix and will be merged soon.
